tests/console/validate_activation_encrypted_partition.pm

```diff
@@ -21,10 +21,12 @@ use base "installbasetest";
 use scheduler 'get_test_suite_data';
 use testapi;
 use validate_encrypt_utils;
+use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
     my $test_data = get_test_suite_data();
     select_console 'install-shell';
+    use_ssh_serial_console if (check_var('BACKEND', 'pvm_hmc'));
```

Comment:
Should this be `check_var('BACKEND', 'pvm_hmc') ? use_ssh_serial_console : select_console 'install-shell' ;`?

Reply:
it works as-is, but we could try.

Reply:
it works, I was doing just like in  logs_from_installation_system, which could be updated as well I guess.